src/core_modules/capture-core/components/Pages/ViewEvent/EventDetailsSection/EventDetailsSection.component.js

```diff
@@ -19,6 +19,9 @@ import { EventChangelogWrapper } from '../../../WidgetEventEdit/EventChangelogWr
 import { OverflowButton } from '../../../Buttons';
 import { ReactQueryAppNamespace } from '../../../../utils/reactQueryHelpers';
 import { CHANGELOG_ENTITY_TYPES } from '../../../WidgetsChangelog';
+import {
+    useMetadataForSingleEventForm,
+} from '../../../DataEntries/SingleEventRegistrationEntry/DataEntryWrapper/hooks/useMetadataForSingleEventForm';
```

Comment:
I know we have huge refactoring job here, but to my knowledge the ViewEvent page is used for both event- and tracker programs. SingleEventRegistrationEntry is only used for event programs. Perhaps we can move this file to `DataEntries/common/ProgramStage` and rename it?